solutions/swb-app/config/jest.config.json

```diff
@@ -1,6 +1,29 @@
 {
   "extends": "@rushstack/heft-node-rig/profiles/default/config/jest.config.json",
   "collectCoverage": true,
+  "transform": {
+    "\\.(ts)$": "ts-jest"
+  },
+  "moduleFileExtensions": ["ts", "js"],
+  "coveragePathIgnorePatterns": [
+    "<rootDir>/src/accountRoutes.ts",
+    "<rootDir>/src/apiRouteConfig.ts",
+    "<rootDir>/src/authRoutes.ts",
+    "<rootDir>/src/costCenterRoutes.ts",
+    "<rootDir>/src/datasetRoutes.ts",
+    "<rootDir>/src/environmentRoutes.ts",
+    "<rootDir>/src/environmentTypeConfigRoutes.ts",
+    "<rootDir>/src/environmentTypeRoutes.ts",
+    "<rootDir>/src/generateRouter.ts",
+    "<rootDir>/src/index.ts",
+    "<rootDir>/src/projectRoutes.ts",
+    "<rootDir>/src/staticPermissionsConfig.ts",
+    "<rootDir>/src/staticRouteConfig.ts",
+    "<rootDir>/src/userRoutes.ts",
+
+    "<rootDir>/src/validatorHelper.ts",
```

Comment:
There was no unit tests in `swb-app` before I added some for audit. Therefore the coverage was close to 0, and I had to ignore these files in test coverage to pass the coverage threshold. I'll create a SIM task to track the files that needs unit test. Unfortunately I do not have enough bandwidth to add unit tests for all of these files in this PR.

Reply:
With the type system verifying our input and output types, I'm not sure there are really that many tests to run on the controllers as long as we keep business logic out of them. My making the controllers "thin" and keeping logic in our services, we can consolidate where all the tests need to live. A unit-testing threshold may not be the best way to tell if we've got sufficient code coverage for `swb-app` like it is in reference and core

Reply:
Yeah for most of the files, I don't think we need unit tests, but for the file listed on line 24 and 25, we could add unit tests for it.

Reply:
For reference, here's the SIM ticket GALI-2122